src/plugins/janus_videoroom.c

```diff
@@ -2205,6 +2206,7 @@ typedef struct janus_videoroom_subscriber_stream {
 	GSList *publisher_streams;						/* Complete list of publisher streams (e.g., when this is data) */
 	int mindex;				/* The media index of this stream (may not be the same as the publisher stream) */
 	char *mid;				/* The mid of this stream (may not be the same as the publisher stream) */
+	char* opaqueid;			/* An id provided while subscribing to uniquely identify the subscription in the updated events */
```

Comment:
Code style: * next to the pointer, not the type.

Reply:
fixed